src/index.js

```diff
@@ -1,7 +1,168 @@
+    markerPosition: { lat: 44.5588,lng: -72.5778 },
+    mapCenter: {lat: 44.5588, lng: -72.5778},
+    town: {}  
+  };
+  
+  startGame = () => {
+    document.getElementById("start-button").disabled = true
+    document.getElementById("guess-button").disabled = false
+    document.getElementById("quit-button").disabled = false
+    let randoLat = this.getRandoLat()
+    let randoLng = this.getRandoLng()
+    let layerArray = leafletPip.pointInLayer ([randoLng, randoLat],L.geoJSON(borderData))
+    while (layerArray.length === 0){
+      randoLat = this.getRandoLat()
+      randoLng = this.getRandoLng()
+      layerArray = leafletPip.pointInLayer ([randoLng, randoLat],L.geoJSON(borderData))
+    }
+    this.setState({
+      mapCenter:{
+        lat: randoLat,
+        lng:randoLng
+      },
+      markerPosition:{
+        lat: randoLat,
+        lng:randoLng
+    }
+  })
+  }
+  componentDidMount = () => {
+    document.getElementById("start-button").disabled = false
+    document.getElementById("guess-button").disabled = true
+    document.getElementById("quit-button").disabled = true
+  }
+  getRandoLat = () => {
+    let lat = Math.random() * (45.005419 - 42.730315) + 42.730315;
+    return lat;
+  }
+ 
+  getRandoLng = () => {
+    let lng = (Math.random() * (71.510225 - 73.35218) + 73.35218) * -1;
+    return lng;
+  }
+
+  showLocation = () => {
+    document.getElementById('latLng').textContent = `Lat: ${this.state.markerPosition.lat} Lng: ${this.state.markerPosition.lng}`
+    document.getElementById('start-button').disabled = false
+    this.lookupLocation()
+  }
+
+  lookupLocation = () => {
+    fetch(`https://nominatim.openstreetmap.org/reverse/?lat=${this.state.markerPosition.lat}&lon=${this.state.markerPosition.lng}&format=json`)
+    .then(response => {
+      return response.json()
+    }) .then(obj => {
+      console.log(obj)
+      document.getElementById('town').textContent = `Town: ${obj.address.village||obj.address.city||obj.address.hamlet}`
+      document.getElementById('county').textContent = `County: ${obj.address.county}`
+    })
+
+  }
+  
+  
+  moveNorth = () => {
+    const lat = this.state.mapCenter.lat
+    const lng = this.state.mapCenter.lng
+    this.setState({
+      mapCenter: {
+        lat: lat + .002,
+        lng: lng,
+      }
+    })
+  }
+  moveSouth = () => {
+    const lat = this.state.mapCenter.lat
+    const lng = this.state.mapCenter.lng
+    this.setState({
+      mapCenter: {
+        lat: lat - .002,
+        lng: lng,
+      }
+    })
+  }
+  moveEast = () => {
+    const lat = this.state.mapCenter.lat
+    const lng = this.state.mapCenter.lng
+    this.setState({
+      mapCenter: {
+        lat: lat,
+        lng: lng + .002
+      }
+    })
+  }
+  moveWest = () => {
+    const lat = this.state.mapCenter.lat
+    const lng = this.state.mapCenter.lng
+    this.setState({
+      mapCenter: {
+        lat: lat,
+        lng: lng - .002,
+      }
+    })
+  }
+
+  return = () => {
```

Comment:
I would recommend renaming this method to something other than "return" since that's a reserved keyword in JavaScript.